content/docs/services/redis28.md

```diff
@@ -14,7 +14,7 @@ cloud.gov offers [Redis](https://www.redis.io/) 2.8 as a service. **This service
 
 Plan Name | Description | Price
 --------- | ----------- | -----
-`standard` | Redis instance with unlimited storage | Free in Alpha
+`standard` | Redis instance with several GB storage | Free in Alpha
```

Comment:
[10 GB](https://gsa-tts.slack.com/archives/cloud-gov-atlas/p1487111719009649) but this number was not selected with any meaningful testing or planning, so while it's in Alpha we want to keep it vague.